core/src/main/java/com/linecorp/armeria/server/logging/LoggingService.java

```diff
@@ -37,72 +40,132 @@
  * @param <I> the {@link Request} type
  * @param <O> the {@link Response} type
  */
+// TODO(anuraag): Make final after removing SampledLoggingService, there isn't a good reason to make this
+// extensible.
 public class LoggingService<I extends Request, O extends Response> extends SimpleDecoratingService<I, O> {
 
-    private static final String REQUEST_FORMAT = "Request: {}";
-    private static final String RESPONSE_FORMAT = "Response: {}";
+    @VisibleForTesting
+    static final String REQUEST_FORMAT = "Request: {}";
+    @VisibleForTesting
+    static final String RESPONSE_FORMAT = "Response: {}";
 
     /**
      * Returns a new {@link Service} decorator that logs {@link Request}s and {@link Response}s at
      * {@link LogLevel#INFO}.
+     *
+     * @see LoggingServiceBuilder for more information on the default settings.
      */
     public static <I extends Request, O extends Response>
     Function<Service<I, O>, LoggingService<I, O>> newDecorator() {
-        return LoggingService::new;
+        return new LoggingServiceBuilder()
+                .requestLogLevel(LogLevel.INFO)
+                .successfulResponseLogLevel(LogLevel.INFO)
+                .failureResponseLogLevel(LogLevel.INFO)
```

Comment:
Sure - it seems like a breaking change though, but a small one. If we go with it, what do you think about just removing `SampledLoggingService` in this PR too?